shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/ui/FeedingPointSheet.kt

```diff
@@ -13,27 +13,30 @@ import androidx.compose.ui.draw.clip
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
 import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.tooling.preview.datasource.LoremIpsum
 import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.feeding.data.model.Feeder
+import com.epmedu.animeal.feeding.data.model.enum.AnimalPriority
+import com.epmedu.animeal.feeding.data.model.enum.Remoteness
+import com.epmedu.animeal.feeding.presentation.model.FeedingPointModel
 import com.epmedu.animeal.foundation.button.AnimealHeartButton
+import com.epmedu.animeal.foundation.common.FeedStatus
 import com.epmedu.animeal.foundation.preview.AnimealPreview
 import com.epmedu.animeal.foundation.switch.AnimalType
 import com.epmedu.animeal.foundation.theme.AnimealTheme
 import com.epmedu.animeal.foundation.theme.CustomColor
-import com.epmedu.animeal.home.data.model.Feeder
-import com.epmedu.animeal.home.data.model.enum.AnimalPriority
-import com.epmedu.animeal.home.data.model.enum.Remoteness
-import com.epmedu.animeal.home.presentation.model.FeedStatus
-import com.epmedu.animeal.home.presentation.model.FeedingPointUi
 import com.epmedu.animeal.resources.R
 import com.mapbox.geojson.Point
 
 @Composable
-internal fun FeedingPointSheetContent(
-    feedingPoint: FeedingPointUi,
+fun FeedingPointSheetContent(
+    feedingPoint: FeedingPointModel,
     contentAlpha: Float,
+    expandToFullScreen: Boolean = false,
```

Comment:
You should use modifier and pass it to the column instead of Spacer depended on this boolean